weldx/time.py

```diff
@@ -228,24 +228,20 @@ class 'TimeDependent' are supported too.
     Direct access and iteration are also supported. The return types are fitting pandas
     types:
 
-    >>> from pandas import Timedelta
-    >>>
-    >>> t = Time(["1s", "2s", "3s"])
-    >>> t[1] == Timedelta(2, "s")
-    True
 
-    >>> isinstance(t[1], Timedelta)
-    True
+    >>> t = Time(["1s", "2s", "3s"])
+    >>> t[1]
+    Time:
+    0 days 00:00:02
 
-    >>> from pandas import Timestamp
-    >>>
     >>> t = Time(["2000", "2001", "2002"])
-    >>> t[1] == Timestamp("2001")
-    True
-
-    >>> isinstance(t[1], Timestamp)
-    True
+    >>> t[1]
+    Time:
+    2001-01-01 00:00:00
+    reference time: 2000-01-01 00:00:00
```

Comment:
Just curious: Have you created those examples with the reprexlite?

Reply:
no I just copied the 'expected' form the doctest error 😉 

but reprexlite makes it super easy, just get rid of `#>` and add the `>>>`, here is the original output
```python
from weldx import Time

t = Time(["2000", "2001", "2002"])
t[1]
#> Time:
#> 2001-01-01 00:00:00
#> reference time: 2000-01-01 00:00:00
```

Reply:
> no I just copied the 'expected' form the doctest error 😉

Ah... a "cheater", I see 😄